items/items-perm.md

```diff
@@ -13,11 +13,19 @@ permalink: /design/access-rights/items/
 1. TOC
 {:toc}
 
-## Preambule on data schema
+## Preamble
+
+Permissions a `Group` has on an `Item` are the result of [**Aggregation**](#aggregation-of-permissions-from-multiple-sources) and [**Propagation**](#aggregation-of-permissions-from-multiple-sources).
 
 Permissions to items given to groups are expressed in the `permissions_granted` table.
-The results of the explicit propagation of permissions through items is stored in the `permissions_generated` table.
-Relations (parent-child) among items have several properties which affect propagation.
+Permissions given to a group are implicitly given to its children.
+This process is called **Aggregation**.
+
+The permissions given for an item may transfer to its children if it is explicitly defined in the parent-child relation.
+Multiple properties exist to define which permissions transfer or not.
+This process is called **Propagation**.
+
+The result of both **Aggregation** and **Propagation** is stored into the `permissions_generated` table.
```

Comment:
No, the aggregation is not stored in the `permissions_generated` table! The aggregation is computed live, the propagation is precomputed into `permissions_generated`.

Reply:
fixed